github3/orgs.py

```diff
@@ -255,6 +255,11 @@ class Organization(BaseAccount):
     # Roles available to members in an organization.
     members_roles = frozenset(['all', 'admin', 'member'])
 
+    def _all_events_url(self, username):
+        url_parts = list(self._uri)
+        url_parts[2] = 'users/{}/events{}'.format(username, url_parts[2])
+        return self._uri.__class__(*url_parts).geturl()
```

Comment:
**tl;dr** This is not in-line with the style of this code base and is unnecessarily cryptic.

The only thing blocking this PR for me is this bit of trickery. I'd rather just import urlparse above and use the ParseResult class directly instead of this. Alternatively,

We can still use `self._build_url` with a `base_url` keyword argument to build it instead of interacting with a ParseResult directly.

If you don't have the time to update this, I'll probably get 'round to it tonight